docs/guides/publish_fetch_packages.md

```diff
@@ -0,0 +1,177 @@
+Similarly as the {{open_aea}} framework, the {{open_autonomy}} framework also works with the concept of **package**: a collection of files that implement a specific component or functionality.
+Publishing a package is simply the process of storing the package into a registry (or in other words, a repository), either locally or remotely.
+
+Remote registries allow to make the package publicly available to other developers. Moreover, it is required that a package be published in a remote registry before [registering the package in the on-chain protocol](./register_packages_on_chain.md).
+
+
+## What will you learn
+In this guide, you will learn to:
+
+  * Manage the life cycle of packages with the framework.
+  * Publish and retrieve packages from local and remote registries.
+
+## How the local and remote registries work
+There are three main types of packages:
+
+  * services,
+  * agents, and
+  * components (which include connections, contracts, protocols and skills).
+
+Services use agent packages, and agents use component packages. Packages are developed locally and stored in a registry to be retrieved and reused in a later stage. Every package has a **public package ID** that follows the naming convention `<author_name>/<package_name>:<version>`.
+
+
+The {{open_autonomy}} framework supports three types of registries:
+
+  * local,
+  * remote [IPFS](https://ipfs.io/), and
+  * remote HTTP.
```

Comment:
we don't support `remote HTTP` atm, so would remove it.

Reply:
fixed